force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -55,10 +70,20 @@ public with sharing class FlowControlLoopsCollections {
      * @return The maximum of the two numbers, or null if either number is null.
      */
     public static Integer findMax(Integer num1, Integer num2) {
-        // Initialize largestNum as null
-        Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        Integer largestNum;
+        if (num1 == num2) {
+            largestNum = num1;
+        }
+        else if (num1 == null || num2 == null) {
+            largestNum = null; // Initialize largestNum as null
+        }
+        else if (num1 > num2) {
+            largestNum = num1;
+        }
+        else if (num1 < num2) {
+            largestNum = num2;
+        }
+        return largestNum; // Replace null with the variable you used to store the result
```

Comment:
Try using the Math class https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_math.htm#apex_System_Math_max